src/services/api.js

```diff
@@ -147,14 +147,19 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             state: 0,
         });
     }
-    sendQuery(query, database, action, stats) {
-        return this.post(this.getPath('/viewer/json/query'), {
-            query,
-            database,
-            action,
-            stats,
-            timeout: 600000,
-        });
+    sendQuery({query, database, action, stats}, {concurrentId} = {}) {
+        return this.post(
+            this.getPath('/viewer/json/query'),
+            {
+                query,
+                database,
+                action,
+                stats,
+                timeout: 600000,
+            },
+            null,
```

Comment:
could you please clarify, what for is this parameter?

Reply:
that's the query params
there is the method signature:
https://github.com/yandex-cloud/axios-wrapper/blob/main/src/index.ts#L231-L236